### PR TITLE
test: use externalized helpers

### PR DESCRIPTION
**Pull request by marionebl**

### CODE REVIEW

File: src/ts-transformer/css-prop/__tests__/index.test.tsx [hunk 1]
```diff
@@ -156,27 +159,25 @@ describe('css prop transformer', () => {
...
-            }}>
-            Hello, world!
-          </div>
-        `,
-        mixins: `
-          export const mixin = { color: 'blue' };
-        `,
-      });
+    it('should transform object with object selector from variable', () => {
+      const actual = transformer.addSource({
```
Comment (madou) [inline src/ts-transformer/css-prop/__tests__/index.test.tsx]: when you add source does it mutate the local above (does every test now have a source?)

File: src/ts-transformer/css-prop/index.tsx [hunk 2]
```diff
@@ -103,7 +103,7 @@ export default function cssPropTransformer(
       // We need to rebind after transforming. Why isn't this in the typed API?
       // @ts-ignore
       ts.bindSourceFile(completedNode, {});
-      console.log(completedNode);
+      // console.log(completedNode);
```
Comment (madou) [inline src/ts-transformer/css-prop/index.tsx]: nice catch we should delete it :D

File: src/ts-transformer/css-prop/__tests__/index.test.tsx (hunk 1 above)
Comment (marionebl) [reply src/ts-transformer/css-prop/__tests__/index.test.tsx]: Currently it does, I'll change the implementation to create new instances for each call to maintain proper encapsulation here.

**Comment by madou** [comment]
hey @marionebl fyi some tests were breaking. the ones that were referencing the `react` module (just made a stub insted) - and then the mixin test (file name was wrong).

easy fix though :)

with react - do you suggest making a mock every time its needed? or should we/can we reference the `node_modules` one?

**Pull request merged**